CHANGELOG.md

```diff
@@ -117,6 +117,7 @@ Every module contains its own CHANGELOG.md. Please refer to the module you are i
 
 ### API Breaking Changes
 
+* (baseapp) [#20361](https://github.com/cosmos/cosmos-sdk/pull/20361) `AppVersionModifier` moved to `core`.
```

Comment:
it was never released but introduced in #16244, so I don't think we need this.

Reply:
In that case I may also rename to not include the package name `app` -> `VersionModifier`